improver_tests/nbhood/square_kernel/test_SquareNeighbourhood.py

```diff
@@ -581,24 +490,20 @@ def test_multiple_times_nan(self):
                 [1.0, 1.0, 1.0, 1.0, 1.0],
             ]
         )
-        cube = set_up_cube(
-            zero_point_indices=((0, 0, 2, 2), (0, 0, 1, 2)),
-            num_time_points=2,
-            num_grid_points=5,
-        )
-        cube.data[0, 0, 0, 0] = np.nan
-        cube.data[0, 1, 1, 1] = np.nan
+        cube = self.multi_cube
```

Comment:
And again - could you rename this test and docstring please.

Reply:
Done.